package.json

```diff
@@ -133,6 +133,6 @@
     "logo": "https://opencollective.com/jest/logo.txt"
   },
   "engines": {
-    "node": ">= 10.14.2"
+    "node": "^10.13.0 || ^12.13.0 || ^14.14.0 || >=15.0.0"
```

Comment:
why 12.13 and 14.14?

Reply:
>match the first version of node 10, 12 and 14 marked as LTS

Reply:
14 will be 14.15 when that's released as LTS next week